change/@internal-react-components-5e7b0e61-f149-4204-9438-9428ca7f6143.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Introduce announcer component which is to be used for announcing when actions are made  ",
```

nit: as this is for the changelog, the part customers will care about is now some control bar buttons announce to screen readers state change
```suggestion
  "comment": "Announce when microphone is turned on/off and video is turned on/off to screen readers and narrator tools",
```